hack/reference-docs-gen-config.json

```diff
@@ -35,6 +35,14 @@
             "typeMatchPrefix": "^knative\\.dev/networking/pkg/apis/networking",
             "docsURLTemplate": "https://pkg.go.dev/knative.dev/networking/pkg/apis/networking#{{.TypeIdentifier}}"
         },
+        {
+            "typeMatchPrefix": "^github\\.com/triggermesh/triggermesh/pkg/apis/common",
+            "docsURLTemplate": "https://pkg.go.dev/github.com/triggermesh/triggermesh/pkg/apis/common/{{arrIndex .PackageSegments -1}}#{{.TypeIdentifier}}"
+        },
```

Comment:
This rule generates links such as:

```html
<code>certKey</code></br>
<em>
<a href="https://pkg.go.dev/github.com/triggermesh/triggermesh/pkg/apis/common/v1alpha1#ValueFromField">
github.com/triggermesh/triggermesh/pkg/apis/common/v1alpha1.ValueFromField
</a>
</em>
```

(The link is currently invalid because the latest TriggerMesh release doesn't contain `pkg/apis/common`. We'll need to wait until the next release before the link becomes valid.)